js/src/logger.ts

```diff
@@ -1276,10 +1276,14 @@ function validateAndSanitizeExperimentLogPartialArgs(
   }
 
   if (event.metrics) {
-    for (const key of Object.keys(event.metrics)) {
+    for (let [key, value] of Object.entries(event.metrics)) {
```

Comment:
why use `let` here instead of `const`?

Reply:
Note that we only use `let` in the scores loop above because we're re-assigning `score`. But we don't need to do that here, so `const` would be better.

Reply:
updated